daycare/migrations/0005_image_data_migration.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 4.0.4 on 2022-04-11 19:07
+
+from django.db import migrations, transaction
+import random
+from daycare.resources.daycare_static_users import DAYCARE_IMAGE_LIST
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ('daycare', '0004_static_daycare_users'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from daycare.models import DayCare, Image
+
+        test_data = []
+
+        for daycare in DayCare.objects.all():
+            for i in range(1, 5):
+                test_data.append((random.choice(DAYCARE_IMAGE_LIST), daycare))
```

Comment:
I would avoid using anything random in test data... it should be explicit.

Reply:
@Omeramsc you can take another look, I changed it that the static images will be more explicit